src/docs-ui/app/main/main.component.ts

```diff
@@ -4,6 +4,8 @@ import { VantageToastService, VantageErrorService } from '@td-vantage/ui-platfor
 import { VantageSessionService } from '@td-vantage/ui-platform/auth';
 import { IUser } from '@td-vantage/ui-platform/user';
 import { timeout } from 'rxjs/operators';
+import { IApp, VantageAppsService } from '@td-vantage/ui-platform/app';
```

Comment:
I know its not part of this PR, but can we rename this to `VantageAppService`?

Reply:
I went back and forth on whether to name it VantageAppService or VantageAppsService.  Should she rename  the files to `app.service.spec.ts` and `app.service.ts` too?